src/modules/mana/sagas.ts

```diff
@@ -14,19 +26,113 @@ import {
   SEND_MANA_REQUEST,
 } from './actions'
 import { Eth } from 'web3x-es/eth'
+import { abiCoder } from 'web3x-es/contract/abi-coder'
 import { ERC20 } from '../../contracts/ERC20'
 import { Address } from 'web3x-es/address'
-import { MANA_CONTRACT_ADDRESS } from './utils'
+import {
+  MANA_CONTRACT_ADDRESS,
+  ERC20_PREDICATE,
+  ROOT_CHAIN_MANAGER,
+} from './utils'
 import { toWei } from 'web3x-es/utils'
 import { getAddress } from 'decentraland-dapps/dist/modules/wallet/selectors'
 import { closeModal } from 'decentraland-dapps/dist/modules/modal/actions'
+import { RootChainManager } from '../../contracts/RootChainManager'
 
 export function* manaSaga() {
+  yield takeEvery(DEPOSIT_MANA_REQUEST, handleDepositMana)
+  yield takeEvery(GET_APPROVED_MANA_REQUEST, handleGetApprovedMana)
```

Comment:
lets add `Request` to the end of the handler name, to keep it consistent with the rest of the codebase, ie: `handleDepositManaRequest` and `handleGetApprovedManaRequest`. This is to keep it different from the `Success` or `Failure` handlers.

Reply:
Fixed! thank you! Good catch!